package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "babel-brunch": "~6.0.0",
     "babel-preset-es2015": "^6.6.0",
+    "babel-preset-es2016": "^6.16.0",
```

Comment:
fwiw, before adding this to the dev dependencies, I also tried the following steps, in hopes it would fix the issue instead:
```
rm -rf ~/.npm
rm -rf node_modules/
npm install
```